auditlog/tests/test_auditlog.py

```diff
@@ -92,12 +92,14 @@ def test_LogCreation4(self):
         groups_vals = [
             {'name': 'testgroup1'},
             {'name': 'testgroup3'},
-            {'name': 'testgroup2'},
+            {'name': 'testgroup2', 'users': self.env['res.users']},
+            {'name': 'testgroup4', 'users': self.env.user},
```

Comment:
It looks like passing recordsets to create is not actually supported:
```
>>> group = self.env['res.groups'].create({'name': 'bla', 'users': self.env.user})
2021-03-17 09:11:54,572 793620 WARNING 14nl odoo.models: Comparing apples and oranges: res.users(1,) == 4 (/home/odoo/buildout/14.0/parts/odoo/addons/mail/models/res_users.py:173
```
```
>>> partner = self.env.user.partner_id
>>> user = self.env['res.users'].create({'login': 'test', partner_id: partner})
Traceback (most recent call last):
  File "<console>", line 1, in <module>
NameError: name 'partner_id' is not defined
>>> user = self.env['res.users'].create({'login': 'test', 'partner_id': partner})
2021-03-17 09:10:05,222 793620 ERROR 14nl odoo.sql_db: bad query: INSERT INTO "res_users" ("id", "create_uid", "create_date", "write_uid", "write_date", "active", "company_id", "login", "notification_type", "partner_id", "signature") VALUES (nextval(%s), %s, %s, %s, %s, %s, %s, %s, %s, %s, %s) RETURNING id
ERROR: can't adapt type 'res.partner'
```
So that would make this bug invalid.

Reply:
Sorry @PieterPaulussen for being late.
Indeed, we must not use recordset in a dict parameter of `write` or `create`.

Reply:
@sebalix Thanks for the feedback. @StefanRijnhart I believe this renders this PR invalid? It must of been a bug in another module.

Reply:
Yes, exactly!